pkgs/applications/science/biology/trimmomatic/default.nix

```diff
@@ -0,0 +1,57 @@
+{ ant
+, fetchFromGitHub
+, jdk11_headless
+, jdk8_headless
+, jre_minimal
+, lib
```

Comment:
Put lib and stdenv first. They will never change, so they will never appear in the diffs.